src/agent/sqlite_manager/src/sqlite_manager.cpp

```diff
@@ -0,0 +1,210 @@
+#include <sqlite_manager.hpp>
+
+#include <format>
+#include <iostream>
+
+namespace sqlite_manager
+{
+    SQLiteManager::SQLiteManager(const std::string& dbName)
+        : m_dbName(dbName)
+        , m_db(make_unique<SQLite::Database>(dbName, SQLite::OPEN_READWRITE | SQLite::OPEN_CREATE))
+    {
+        // Open the database in WAL mode
+        m_db->exec("PRAGMA journal_mode=WAL;");
+    }
+
+    int SQLiteManager::CreateTable(const std::string& tableName, const std::vector<Col>& cols)
```

Comment:
If we launch an exception in case of error, I think a return value is not necessary.

Reply:
Done. Function now returns void.